src/side-nav.scss

```diff
@@ -72,6 +72,10 @@ $has-child-content-arrow-margin: 0.125rem;
   &--condensed {
     width: 2.75rem;
 
+    &--popover-custom {
```

Comment:
can't add example for this as we are not showing interaction on side-nav click. and this is modifier for `fd-popover-custom` class.

Reply:
You can by hardcoding the popover body to be always open. There are multiple examples in the documentation. You can check Action Bar `Contextual` example.

Reply:
And you can always add some JavaScript for the documentation purposed. You can check Vertical Navigation for exactly the same case: https://fundamental-styles.netlify.app/?path=/docs/components-vertical-navigation--cozy

Reply:
i do not see any change on using `display: block` instead of using `display: inline-block` here. I think change in Ngx can be safely removed without migrating to Styles.

with `display: block`
<img width="1676" alt="Screenshot 2021-07-13 at 5 34 41 PM" src="https://user-images.githubusercontent.com/57661487/125449432-27da07ca-6a63-4fbb-8def-9373038a2124.png">

with `display: inline-block`
<img width="1672" alt="Screenshot 2021-07-13 at 5 34 59 PM" src="https://user-images.githubusercontent.com/57661487/125449485-32be76d8-202e-4a91-aa80-3cbdd6c4aff7.png">

Hence this PR can be closed.